src/app/collection/pages/nice-challenge/components/philosophy/philosophy.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="wrapper">
+
+  <div class="philosophy-content">
+    <div class="grid">
+      <div class="grid-item">
+        <img 
+          src="../../../../../../assets/images/Nice-Logo-Black.png"
+          alt="NICE Challenge Project Logo"
+          class="logo-img"
+        >
+      </div>
+      <div class="grid-item">
+        <p>
+          Hosted at California State University at San Bernardino (CSUSB), 
+          the NICE Challenge Project provides competency-based assessments 
+          focused on real-world problems and context, designed to capture 
+          useful data for actionable metrics.
+        </p>
+      </div>
+      <div class="grid-item">
+        <p>
+          The National Initiative for Cybersecurity Education (NICE) 
+          Challenge Project develops real-world cybersecurity challenges 
+          within virtualized business environments that bring students the 
+          experience before joining the workforce.
+        </p>
+      </div>  
+      <div class="grid-item">
+        <img 
```

Comment:
Could we use a darker logo for them? Although, contrast doesn't really matter for images it kind of blends into the background more than in the mocks.